Libs/RaptureCommon/src/main/java/rapture/common/impl/jackson/JacksonUtil.java

```diff
@@ -86,22 +94,37 @@
         }
     }
 
+    /**
+     * Use jsonFromObject(object, prettyfy)
+     * 
+     * @param object
+     * @return
+     */
+    @Deprecated
     public static String jsonFromObject(Object object) {
-        StringWriter writer = new StringWriter();
+        return jsonFromObject(object, false);
+    }
+
+    public static String jsonFromObject(Object object, boolean pretty) {
```

Comment:
Can you simply create a prettyfy named function and call this with that flag. When you use this function later on the boolean parameter and its effect is completely obscure...

i.e..

prettyJson(Object object) { return jsonFromObject(object, true); }